internal/k8s/kctl/deploy.go

```diff
@@ -19,12 +21,23 @@ type Wait interface {
 	Process(labels []string)
 }
 
+func pause() {
```

Comment:
Were there any race conditions that you ran into? We can implement wait to handle these scenarios too. I haven't run into issues so far.

Reply:
I and @alfsuse had some problem with some plugin in k3ai, we put a pause and we solved. 
This is why you see the [pauses](https://github.com/kf5i/k3ai-plugins/blob/main/plugin_jupyter-scipy/install#L114)  on the plugins. (We haven't investigated in detail btw)

Reply:
ok. I am cool with this for now. We should revisit this when we implement wait.